examples/infrastructure_as_code/oneview-config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "ip": "10.50.9.90",
```

Comment:
As a good practice, please mention like <property_name> instead of giving values. Follow the same for all other properties in this config.

Reply:
removed this file and made changes in playbooks to use the oneview_config.json file in examples folder